qemu/tests/qemu_disk_img_snapshot.py

```diff
@@ -6,7 +6,7 @@
 class SnapshotTest(qemu_disk_img.QemuImgTest):
 
     def __init__(self, test, params, env):
-        self.tag = params.get("image_snapshot", "image1")
+        self.tag = params.get("images", "image1")
```

Comment:
images is a list of image need to prepare for testing, so directly use images not suitable.